pkg/agent/externalnode/external_node_controller.go

```diff
@@ -588,6 +588,16 @@ func (c *ExternalNodeController) removeOVSPortsAndFlows(interfaceConfig *interfa
 		}
 	}()
 
+	// This is for issue #5111 (https://github.com/antrea-io/antrea/issues/5111), which may happen if an error occurs
+	// when moving the configuration back from host internal interface to uplink. This logic is run in the second
+	// try after the error is returned, at this time the host internal interface is already deleted, and the uplink's
+	// name is recovered. So the ips and routes in "adapterConfig" are actually read from the uplink and no need to
+	// move the configurations back. The issue was seen on VM with RHEL 8.4 on azure cloud.
+	if !hostInterfaceExists(uplinkIfName) {
+		klog.InfoS("Uplink is not existing on the host, return", "uplinkIfName", uplinkIfName)
```

Comment:
```suggestion
		klog.InfoS("The interface with uplink name did not exist on the host, skipping its recovery", "uplinkIfName", uplinkIfName)
```
to avoid ambiguity (It's not that uplink doesn't exist) and grammar issue.

Reply:
updated.